apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransaction.cs

```diff
@@ -38,6 +38,7 @@ public sealed class SpannerTransaction : DbTransaction, ISpannerTransaction
         private readonly List<Mutation> _mutations = new List<Mutation>();
         private DisposeBehavior _disposeBehavior = DisposeBehavior.ReleaseToPool;
         private bool _disposed = false;
+        private bool _used = false;
```

Comment:
Could you either give this a more specific name, or add a comment to clarify its exact meaning?